### PR TITLE
Validate module

### PR DESCRIPTION
**Pull request by tonyjhuang**

### CODE REVIEW

**Comment by davlin93** [comment]
+1

**Comment by kehphin** [comment]
-1

**Comment by kehphin** [comment]
youre at 0 now

**Comment by yangk** [comment]
asdfasdf

**Comment by kehphin** [comment]
aaaa

File: js/validate.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  // Validates the other user defined input params ansd makes sure
+  // they make sense/are possible.
+  //
+  // Creation date: 3/1/15
+  // Modifications list:
+  //
+  my.validateNumArgs = function(numGroups, numPictures, pictures) {
+    var size = _.size(pictures);        
+    if(numGroups * numPictures > size) {
+        throw new Error("Can't split " + size + " total pictures into "
```
Comment (kehphin) [inline js/validate.js]: WHAT THE FUCK IS THIS BULLSHIT

**Comment by kehphin** [comment]
plus one

**Pull request merged**